keyboards/fjlabs/block67hs/config.h

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021 <felix@fjlaboratories.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x7074 // FJLabs
+#define PRODUCT_ID   0x1021 // Block67 Hotswap
+#define DEVICE_VER   0x0001 // Version 1
+#define MANUFACTURER    FJLabs
```

Comment:
You might want to re-read the PR checklist, as this doesn't even compile against master (and wont have done for a fair while now).

Reply:
**sigh**, pushing 2 year old code up always begets some problems. I'm still running a much older QMK locally, so I suppose that's why it compiled perfectly here. 

Will update for this new `info.json`...